src/main/java/org/breedinginsight/brapps/importer/services/processors/GermplasmProcessor.java

```diff
@@ -175,12 +215,13 @@ public void getExistingBrapiData(List<BrAPIImport> importRows, Program program)
                 .newObjectCount(newGermplasmList.size())
                 .build();
 
+        //Modified logic here to check for female parent dbid or entry no, removed check for male due to assumption that shouldn't have only male parent
```

Comment:
I don't think this assumption is correct.  Some of the breeding methods allow for a known male, but not female, or multiple male parents (because it may be open pollination, and you don't know the exact male, just that it came from a group).  May be worth talking with @shawnyarnes on this.

Reply:
I had asked Shawn before about whether there could be a male parent but not a female, but you make a good point that the _known_ qualifier might have impact. I'll ask.